contracts/CourtPresaleActivate.sol

```diff
@@ -62,6 +155,21 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
         bondedToken.approve(address(registry), bondedTokensObtained);
         registry.stakeFor(_from, bondedTokensObtained, abi.encodePacked(ACTIVATE_DATA));
 
+        // refund leftovers if any
+        _refund(bondedToken);
+
         emit BoughtAndActivated(_from, _token, _amount, bondedTokensObtained);
     }
+
+    function _refund(address _tokenAddress) internal {
```

Comment:
Do we really need this extra jump? I think asking for a cast at the call site should be enough?

Reply:
With the change above, not anymore :wink: